### DISCUSSION TITLE
Sensor value to use in pages.json

### DISCUSSION THREAD

**Discussion by sbruynin2340** [HASwitchPlate/openHASP#732] [Q&A] [answered]
I captured a sensor value with my_custom.cpp and want to show it in a specific page on the display. Can this be done via pages.json? Can the sensor value be linked to a page / id?

**Comment by fvanroie** [comment, marked as answer]
You need to do this in your custom code. You can use openHASP or LVGL api's to modify the object:
check out https://github.com/HASwitchPlate/openHASP/discussions/653

**Reply by sbruynin2340** [reply]
Thanks, it works. Great firmware.
I now ran into a problem: I uploaded a binary font (probably to big) and get :
Prompt > [       3.158][102388/112452  8][19620/35488 45] LVGL: Couldn't allocate memory
Prompt > [       3.168][102388/112516  9][19620/35488 45] LVGL: lv_font_get_bitmap_fmt_txt
Prompt > [       3.179][98292/110388 10][19620/35488 45] LVGL: Out of memory (0x00000000)
and I cannot access the webapp to delete the font file. Any other way that gives access to the filesystem?

**Comment by sbruynin2340** [comment]
I solved the memory problem by increasing LV_MEM_SIZE and LV_VDB_SIZE in include/user_config_override.
Access to the webapp was possible again and then delete the large fontfile using the file editor.
Thanks anyway for your help.
Stan
